app/javascript/src/js/admin/queue_status_submit.js

```diff
@@ -19,7 +19,7 @@ import domready from 'domready';
 
 domready(function() {
   function notOurForm(form) {
-    return !(form && form["data-auto-submit"] == "true");
```

Comment:
form["data-auto-submit"] wasn't a good expression for getting that data attribute. I just created a new one just for the purposes of identifying the form.

Reply:
I don't understand what made it not a good expression. Can you say more?